### DISCUSSION TITLE
Iris compatible?

### DISCUSSION THREAD

**Discussion by kant** [MonitorControl/MonitorControl#757] [Q&A]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Is compatible with [Iris](https://iristech.co/)?

**Comment by waydabber** [comment]
I don't know, never heard of Iris until now. :) If it works like f.lux then it is compatible depending on the dimming mode used. Please let me know your findings!

I'll move this over to Discussions as a Q&A item. Let's hope for an answer!
